<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatusWidget</class>
 <widget class="QWidget" name="StatusWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="Vec3Widget" name="widgetMPos" native="true"/>
   </item>
   <item>
    <widget class="Vec3Widget" name="widgetWPos" native="true"/>
   </item>
   <item>
    <widget class="Vec3Widget" name="widgetWCO" native="true"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelAccesorryState">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditAccessoryState"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelBufferState">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditBufferState"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelFeedRate">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditFeedRate"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelSpindleSpeed">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditSpindleSpeed"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelLineNumber">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEditLineNumber"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelMachineState">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEditMachineState"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelOverriddenValues">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="lineEditOverridenValues"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelInputPinState">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEditInputPinState"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Vec3Widget</class>
   <extends>QWidget</extends>
   <header>vec3/Vec3Widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
